lib/airbrake.js

```diff
@@ -46,7 +47,7 @@ function Airbrake() {
   ];
 }
 
-_.assign(Airbrake.prototype, EventEmitter.prototype);
+assign(Airbrake.prototype, EventEmitter.prototype);
```

Comment:
I don't think performance really matters here. May I ask you to use only `merge`, given that we don't use `assign` anywhere else?